### DISCUSSION TITLE
Self-host premium plugins?

### DISCUSSION THREAD

**Discussion by finnar-bin** [tinymce/tinymce#9160] [General]
Is there a way to self-host the premium plugins if that's even a thing? I'm following the steps [here](https://www.tiny.cloud/docs/tinymce/6/react-pm-bundle/) to get up and running. Probably load those in via external_plugins or something along those lines?

**Comment by TheSpyder** [comment]
Premium plugins can be self-hosted with our enterprise plan, however we do not offer premium support via GitHub. If you have a support subscription, you will be better served by our support team who will be able to assist: https://support.tiny.cloud/hc/en-us/requests/new

**Reply by finnar-bin** [reply]
Gotcha, thanks!
